src/test/skript/tests/syntaxes/sections/SecParse.sk

```diff
@@ -2,7 +2,7 @@ test "parsing section":
   parse:
     parse:
       # intentionally empty to cause an error
-  assert last parse logs contain "A parse section must contain code" with "Parse section didn't return expected error"
+  assert last parse logs contain "A parse section must contain code" with "Parse section didn't return expected error: %last parse logs%"
```

Comment:
you could implement VerboseAssert on CondContains while you're here

Reply:
What's that 😦